storage/interface.go

```diff
@@ -364,6 +364,7 @@ type DescribeVolumesResult struct {
 type AttachVolumesResult struct {
 	VolumeAttachment *VolumeAttachment
 	Error            error
+	ErrorMsg         error // Refers to body content of Error that provides context to azure error response
```

Comment:
We don't want to expose Azure internals here. The Error value should contain what's needed.

Reply:
Sure thing thanks ian!